openproblems/tasks/_batch_integration/batch_integration_graph/README.md

```diff
@@ -36,7 +36,10 @@ Datasets should contain the following attributes:
 * `adata.obsm['X_uni']` with the PCA embedding of the unintegrated representation
 * `adata.obsp['uni_connectivities']` with an unintegrated connectivity matrix generated
   by  `scanpy.pp.neighbors()`
-* `adata.X` with log-normalized data
+* `adata.X` WARNING: other than most tasks, this should contain log-normalized data.
+   This is the case as we are comparing against the normalized un-integrated data and
+   the data compes from different origins
+* `adata.layers['log_normalized']` with log-normalized data
```

Comment:
Please describe how the data were normalized. log scran for UMIs and TPM for full-length, I think?